common/core/src/main/java/com/vmware/connectors/common/payloads/response/CardAction.java

```diff
@@ -167,12 +170,22 @@ public boolean isRemoveCardOnCompletion() {
      * Returns the "completed label". This is the text to be displayed to the user on successful
      * completion of the action.
      *
+
      * @return the completed label
      */
     public String getCompletedLabel() {
         return completedLabel;
     }
 
+    /**
+     * Returns "allow repeated" value. If set to true, then the client can show
+     * the "OPEN_IN" action irrespective of "completed label".
+     *
```

Comment:
We need to fix this comment. `OPEN_IN` is just an example.
But connectors developers can use the property for any of their card-action.

Reply:
Yes, that's correct.

Reply:
I have made the comment generic now.